### DISCUSSION TITLE
Flood abandoned

### DISCUSSION THREAD

**Discussion by rootdet** [swizzin/swizzin#545] [Feature Requests] [answered]
The version of flood installed by Swizzin is long abandoned. the source files are coming from https://github.com/jfurrow/flood.git  This has not been updated in about 2 years now. A fork of the project has been started and the community is very active - https://github.com/jesec/flood.git

I suggest updating the scripts and/or doing some sort of migration path over.

**Comment by flying-sausages** [comment, marked as answer]
Yeah, it's been something that's on my radarr. I would like to do a general overhaul/rewrite of the entire `flood` installer so that we don't have to compile a session per each user, which is wasteful and needlessly heavy.

If you've got any experience running/installing/comparing the `jesec` fork, let me know here for sure.

NOTE: I converted this to a discussion as a bug report is not the valid format for this. Please read the options you gt before making an issue more closely.
